h11/_headers.py

```diff
@@ -56,6 +56,7 @@
 # Maybe a dict-of-lists would be better?
 
 _content_length_re = re.compile(br"[0-9]+")
+_header_value_re = re.compile(br"[\x21-\xff]+(?:[ \t]+[\x21-\xff]+)*|^$")
```

Comment:
Would it make more sense to do `from ._readers import field_value` instead of duplicating the regex here?